packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -11,6 +13,117 @@ import 'binding.dart';
 import 'message_codec.dart';
 import 'message_codecs.dart';
 
+/// When this is is true statistics about the usage of Platform Channels will
+/// be printed out periodically and Timeline events will show the time between
+/// sending and receiving a message (encoding and decoding time excluded).
+bool debugProfilePlatformChannels = false;
```

Comment:
This should go in `packages/flutter/lib/src/services/debug.dart` instead?

Reply:
Yes - along with the appropriate assertions about it being reset if someone sets it in a test.

Reply:
done